android/app/src/main/java/com/cBreez/client/BreezForegroundService.kt

```diff
@@ -0,0 +1,137 @@
+package com.cBreez.client
+
+import android.app.Service
+import android.content.Intent
+import android.os.Binder
+import android.os.Build
+import android.os.Handler
+import android.os.IBinder
+import android.os.Looper
+import android.os.Parcelable
+import androidx.annotation.RequiresApi
+import com.cBreez.client.BreezNotificationService.Companion.dismissForegroundServiceNotification
+import com.cBreez.client.BreezNotificationService.Companion.notifyForegroundService
+import com.cBreez.client.BreezNotificationService.Companion.registerNotificationChannels
+import com.cBreez.client.Constants.NOTIFICATION_ID_FOREGROUND_SERVICE
+import com.google.firebase.messaging.RemoteMessage
+import org.tinylog.kotlin.Logger
+
+
+class BreezForegroundService : Service() {
+    companion object {
+        private const val TAG = "BreezForegroundService"
+    }
+
+    inner class BreezNodeBinder : Binder() {
+        fun getService(): BreezForegroundService = this@BreezForegroundService
+    }
+
+    private val binder = BreezNodeBinder()
+
+    /** True if the service is running headless (that is without a GUI). In that case we should show a notification */
+    @Volatile
+    private var isHeadless = true
+
+    override fun onCreate() {
+        super.onCreate()
+        Logger.tag(TAG).debug("Creating Breez node service...")
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            registerNotificationChannels(applicationContext)
+        }
+        startForeground(
+            NOTIFICATION_ID_FOREGROUND_SERVICE,
+            notifyForegroundService(applicationContext)
+        )
```

Comment:
We already start the foreground service in onStartCommand . It seems wrong that we need to do it here as well.

Reply:
Done, but removed from onStartCommand instead c5d1e7efd906d0d756e38726b6278a726129f74e

When I removed this from onCreate, the foreground service notification persisted, dismissing programmatically after task ended didn't made any changes as well.